Corona-Warn-App/src/main/assets/privacy_en.html

```diff
@@ -1,9 +1,112 @@
+<h1>
+    Notice on the termination of the processing of personal data
+</h1>
+
+<p>
+    PLEASE NOTE: The following notice applies from 1 May 2023. The privacy notice valid until 30
+    April 2023 can be found below.
+    All previous versions of the privacy notice can be found at: <a>https://www.coronawarn.app/en/privacy</a>
+</p>
+
+<p>
+    <strong>1. Termination of data processing </strong>
+</p>
+<p>
+    On 1 May 2023, the Corona-Warn-App will go into sleep mode. This will reduce the previous range
+    of features. Most features, including the warning feature and test result retrieval, will no
+    longer be available.
+    Data processing on the server system of the Robert Koch Institute (RKI), which is required for
+    these features, will stop when the sleep mode begins. From this point on, the RKI will no longer
+    process any personal data in connection with these features. The personal data stored on the RKI
+    server system for these features will be deleted completely.
+    However, you will still have access to the COVID certificates stored in the Corona-Warn-App, and
+    any entries in the contact journal, for as long as the app is installed on your smartphone. You
+    will not, however, be able to store new COVID certificates in the app and you will not receive
+    any COVID certificate renewal notices. If you want to continue using the stored COVID
+    certificates and the contact journal, please refer to the notes at the end of this section.
+    When sleep mode is activated, the Corona-Warn-App will disable your operating system’s COVID-19
+    exposure notification system, which is used to send out exposure data and record encounters
+    (iPhone: “Exposure Notifications”; Android smartphones: “COVID-19 Exposure Notifications”). You
+    can also disable the COVID-19 exposure notification system yourself at any time in your
+    operating system settings. Your operating system automatically deletes the exposure data stored
+    on your smartphone after 14 days. If you wish to delete it earlier, you can do so manually in
+    your operating system settings or uninstall the Corona-Warn-App.
+    The exchange servers of the countries that participated in the transnational warning system have
+    already been switched off. All personal data processed in connection with the transnational
+    warning system has been deleted.
+    From 1 June 2023, the Corona-Warn-App will not be developed any further.
+    <u>
+        Notes on using the features for digital COVID certificates and the contact journal from 1
+        May 2023:
+    </u>
+</p>
+<ul>
+    <li>
+        You can still use the export features to back up your stored digital COVID certificates and
+        entries in your contact journal. If you wish to present a digital COVID certificate as
+        proof, please note the information in Section Error: Reference source not found of the

```

Comment:
"Section Error: Reference source not found"

is an error

Reply:
@OLVI5

There are multiple issues with the updated texts. Please have a look again at both the iOS PR and the Android PR and make sure that all issues are fixed, also in the PDF that will go live on the website.

Reply:
Hi @Ein-Tim ,
we will look at the proposed changes and go through the documents again. Thank you very much!
Best Oliver